gef.py

```diff
@@ -1263,6 +1266,19 @@ def get_glibc_arena(addr=None):
         return None
 
 
+def get_glibc_arenas(addr=None, get_all=True):
+    arena = get_glibc_arena(addr)
+    arenas = [arena]
+    if not get_all:
+        return arenas
+    while True:
+        arena = arena.get_next()
```

Comment:
For another time: We could make the glibc arena type an iterable (implement `__iter__`) so that we could do `return [arena for arena in iter(arena)]`

Reply:
Good point, might as well do it now ? cc @theguy147

Reply:
You can actually do it pretty easily with a generator. Basically take this code and put it into `__iter__`

```python
def __iter__(self):
    while True:
        yield arena
        arena = arena.get_next()
        is arena is None:
            raise StopIteration
```

Reply:
Good idea. I added this with the latest commit.